lib/open_api_spex/plug/put_api_spec.ex

```diff
@@ -48,4 +44,28 @@ defmodule OpenApiSpex.Plug.PutApiSpec do
     |> Stream.map(fn operation -> {operation.operationId, operation} end)
     |> Enum.into(%{})
   end
-end
\ No newline at end of file
+
+  defp fetch_spec(spec_module) do
+    # This if statement and the contents of the else block can be
+    # removed once an Elixir lang release requires an OTP version >=21.2
+    if function_exported?(:persistent_term, :info, 0) do
+      try do
+        :persistent_term.get(spec_module)
+      rescue
+        ArgumentError ->
+          IO.puts("Warning: OpenApiSpec spec not found in persistent terms")
+          spec = build_spec(spec_module)
+          :ok = :persistent_term.put(spec_module, spec)
+          spec
+      end
+    else
+      case Application.get_env(:open_api_spex, spec_module) do
+        nil ->
+          spec = build_spec(spec_module)
+          Application.put_env(:open_api_spex, spec_module, spec)
+          spec
+        spec -> spec
```

Comment:
Can you apply Elixir formatter just to the additions of this file?

Reply:
It's all or nothing on the entire file.

Reply:
When I was using Atom, the formatter plugin I was using was able to format a selected area. Sadly, that feature is not available in my VSCode.